src/libs/ck-libs/tcharm/tcharm.C

```diff
@@ -11,7 +11,7 @@ Orion Sky Lawlor, olawlor@acm.org, 11/19/2001
 
 CtvDeclare(TCharm *,_curTCharm);
 
-static int lastNumChunks=0;
+static int nChunks;
```

Comment:
Any particular reason why this variable and the static variable below are no initialized?

Reply:
They get implicitly set to zero by the BSS segment.

Reply:
Sure, but why not write it down explicitly?

Reply:
Doesn't that increase the size of the .data segment?

Reply:
I think it does, but no idea if that's important or not. This PR is fine for me either way.